keps/sig-api-machinery/0006-apply.md

```diff
@@ -372,6 +376,69 @@ updated when we know the concrete things changing for beta.
 This will be promoted to GA once it's gone a sufficient amount of time as beta
 with no changes. A KEP update will precede this.
 
+### Upgrade / Downgrade Strategy
+
+#### Upgrade from kubectl Client-Side to Server-Side Apply
+
+With client-side `kubectl apply`, the annotation
+`kubectl.kubernetes.io/last-applied-configuration` tracks ownership for a single
+shared field manager. With server-side `kubectl apply --server-side`, the
+`.metadata.managedFields` field tracks ownership for multiple field managers.
+
+Users who wish to start using server-side apply for objects managed with
+client-side apply would encounter a field manager conflict: the field set that
+the user now wants to manage with server-side apply will be owned by the
+client-side apply field manager.
+
+In this case, users would need to force conflicts with `kubectl apply
```

Comment:
It's not clear that this is a problem we seek to avoid, I'd reword to add something like "If we don't specifically handle this case, then".